utils/get_supported_scylla_base_versions.py

```diff
@@ -15,7 +15,7 @@
 
 
 # We support to migrate from specific OSS version to enterprise
-supported_src_oss = {'2023.1': '5.2', '2022.2': '5.1', '2022.1': '5.0', '2021.1': '4.3', '2020.1': '4.0'}
+supported_src_oss = {'2023.2': '5.3', '2023.1': '5.2', '2022.2': '5.1', '2022.1': '5.0', '2021.1': '4.3'}
```

Comment:
can you please reverse this dict, that new versions would come at the end ?

Reply:
done